src/gluonts/trainer/_base.py

```diff
@@ -57,12 +58,6 @@ def loss_value(loss: mx.metric.Loss) -> float:
     return loss.get_name_value()[0][1]
 
 
-class BestEpochInfo(NamedTuple):
```

Comment:
Why removing this class? If I understand correctly, it’s being replace by a dictionary. I thought we rename this into “EpochInfo”, but maybe there are reasons to remove it instead?

Reply:
NamedTuple is not json serializable. The structure is destroyed and when you deserialize you get back just a list. Dictionaries on the other hand do not have this issue.